content/en/docs/Writing policies/variables.md

```diff
@@ -406,11 +406,21 @@ regex_replace_all_literal(regex string, src string|number, replace string|number
 regex_match(string, string|number) bool
 pattern_match(pattern string, string|number) bool ('*' matches zero or more alphanumeric characters, '?' matches a single alphanumeric character)
 label_match(object, object) bool (object arguments must be enclosed in backticks; ex. `{{request.object.spec.template.metadata.labels}}`)
+
 add(number, number) number
+add(quantity|number, quantity|number) quantity (returns a quantity if any of the parameters is a quantity)
+add(duration|number, duration|number) duration (returns a duration if any of the parameters is a duration)
 subtract(number, number) number
+subtract(quantity|number, quantity|number) quantity (returns a quantity if any of the parameters is a quantity)
+subtract(duration|number, duration|number) duration (returns a duration if any of the parameters is a duration)
 multiply(number, number) number
-divide(number, number) number (divisor must be non zero)
-modulo(number, number) number (divisor must be non-zero, arguments must be integers)
+multiply(quantity|number, quantity|number) quantity (returns a quantity if any of the parameters is a quantity)
+multiply(duration|number, duration|number) duration (returns a duration if any of the parameters is a duration)
+divide(quantity|number, quantity|number) quantity|number (returns a quantity if exactly one of the parameters is a quantity, else a number; the divisor must be non-zero)
```

Comment:
Can we put a note here, given recent discussions, that users need to specify inputs in the correct format given the expected input type?

"An input expected to be an integer/number must be wrapped in back ticks while quantity/string should be specified wrapped in single quotes." <== something like this?